f/fcs/fn_fcsInit.sqf

```diff
@@ -24,83 +24,93 @@
 
 */
 
-// Doesn't need to run on DS
-if (isDedicated) exitWith {};
+// Doesn't need to run on DS or HC
+if !(hasInterface) exitWith {};
 
-params ["_vehicle"];
+params ["_vehicle",["_mode",true]];
 
 if !(_vehicle isKindOf "LandVehicle") exitWith { diag_log "FCS: tried to run on something that isn't a vehicle" };
 	
-if (_vehicle getVariable ["fcs_hasEH",false]) exitWith { diag_log "FCS: tried to run on something that already has FCS set up"};
+if (_vehicle getVariable ["f3_fcs_hasEH",false]) exitWith { diag_log "FCS: tried to run on something that already has FCS set up"};
 
-// Commander's override action
-_vehicle addAction
-[
-	"Commander's override",	
-	{
-		params ["_target", "_caller", "_actionId", "_arguments"];
-		[_target,_caller] call f_fnc_fcsCommanderOverride;
-	},
-	nil,	
-	1.5,	
-	false,	
-	true,	
-	"",	
-	"(_this == commander _target) && {!(isNull gunner _target) && !(_target getVariable [""fcsCommanderOverride_cooldown"",false]) && !(_target getVariable [""fcs_failure"",false])}", 
-	0,		
-	false,	
-	"",	
-	""	
-];
+// If the FCS briefing tab hasn't been added already, add it.
+if (isNil "f3_fcs_briefingDone") then {
+	[] call f_fnc_fcsBriefing;
+};
 
+// Only do this part in the default mode. If in the alternate mode, skip this and only add the driver action.
+if (_mode) then {
```

Comment:
At this point, would the driver actions not be better as a separate component? Adding the `_mode` makes this feel a little overloaded (and would remove the need for the `Note:` in the briefing tab)